src/read/read_ref.rs

```diff
@@ -65,23 +62,28 @@ where
     /// Note that not all writes will be included with this read -- only those that have been
     /// refreshed by the writer. If no refresh has happened, or the map has been destroyed, this
     /// function returns `None`.
-    pub fn get<'a, Q: ?Sized>(&'a self, key: &'_ Q) -> Option<&'a Values<V, S>>
+    pub fn get<Q: ?Sized>(&self, key: &Q) -> Option<&Values<V, S>>
```

Comment:
This change is not okay, because it means the return value is now also tied to the lifetime of the provided key.